File: tests/expected/cast.go
package main

import (
	"fmt"
)

func MainFunc() {
	var a int16 = int16(1)
	b := a
	fmt.Printf("%v\n", b)
	var c int64 = int64(1)
	d := c
	fmt.Printf("%v\n", d)
	var e uint64 = uint64(1)
	f := e
	fmt.Printf("%v\n", f)
}

func main() {
	MainFunc()
}

File: tests/expected/cast.nim

proc main_func() =
  let a = int16(1)
  let b = a
  echo b
  let c = int64(1)
  let d = c
  echo d
  let e = uint64(1)
  let f = e
  echo f

proc main() =
  main_func()

main()
